lddtree.sh

```diff
@@ -25,6 +25,7 @@ usage() {
 	  -R <root>       Use this ROOT filesystem tree
 	  --no-auto-root  Do not automatically prefix input ELFs with ROOT
 	  --no-recursive  Do not recursivly parse dependencies
+	  --no-header     Do not show header (binary and interpreter info)
```

Comment:
How does it differ from parsing `readelf -d | grep NEEDED`?

eg. `readelf -d $_path | grep '(NEEDED)' | sed -e 's/.*\[//' -e 's/\].*//'`